Gemfile

```diff
@@ -67,7 +67,8 @@ else
 end
 
 gem 'sass', '3.4.9'
-gem 'sass-rails'
+gem 'sass-rails' # required by bootstrap-sass, but not listed as a dependency of it
+gem 'sassc-rails', github: 'bolandrm/sassc-rails', branch: 'master'
```

Comment:
Forgive my possible misunderstanding on how Gemfiles and Gemfile.locks work but if the master branch changes on this repo is there a chance that we will get that when we bundle? Would it be safer to point this at a sha rather than a branch?